### PR TITLE
vendor: upgrade cobra 0.0.5->1.1.3

### PR DESCRIPTION
**Pull request by cucaroach**
This is the result of doing the following:
```shell
go get -u github.com/spf13/cobra
go mod tidy
make vendor_rebuild
make bazel-generate
````

This caused lots of changes to happen that I don't completely grok so I may be out of my depth here.   I also don't understand the teamcity build failures.   Help!   Happy to throw this away, mostly it was a learning exercise for me.

### CODE REVIEW

**Comment by cucaroach** [comment]
I tried and failed to link this PR to issue 50187, not sure what I'm doing wrong

**Comment by otan** [comment]
you'll need to commit the vendor repo as well:
```
cd vendor
git checkout -b branch_name
git push
```

as per https://github.com/cockroachdb/cockroach/blob/master/build/README.md#master-branch-pointer-in-vendored-repo

**Comment by cucaroach** [comment]
I think I did that: https://github.com/cockroachdb/vendored/tree/treilly-cobra-113

**Comment by otan** [comment]
I see.

the errors:
```

[15:54:24][Compile C dependencies] go: inconsistent vendoring in /go/src/github.com/cockroachdb/cockroach:
[15:54:24][Compile C dependencies] 	github.com/DataDog/datadog-go@v4.4.0+incompatible: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
[15:54:24][Compile C dependencies] 	github.com/bazelbuild/rules_go@v0.26.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
[15:54:24][Compile C dependencies] 	github.com/cockroachdb/errors@v1.8.3: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
[15:54:24][Compile C dependencies] 	github.com/cockroachdb/pebble@v0.0.0-20210313162627-639dfcee1d23: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
[15:54:24][Compile C dependencies] 	github.com/klauspost/compress@v1.11.12: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
[15:54:24][Compile C dependencies] 	github.com/lightstep/lightstep-tracer-go@v0.24.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
[15:54:24][Compile C dependencies] 	golang.org/x/exp@v0.0.0-20210220032938-85be41e4509f: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
[15:54:24][Compile C dependencies] 	golang.org/x/sys@v0.0.0-20210316092937-0b90fd5c4c48: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
[15:54:24][Compile C dependencies] 	gopkg.in/DataDog/dd-trace-go.v1@v1.28.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
[15:54:24][Compile C dependencies] 	github.com/StackExchange/wmi@v0.0.0-20180116203802-5d049714c4a6: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
[15:54:24][Compile C dependencies] 	github.com/cockroachdb/errors@v1.8.2: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
[15:54:24][Compile C dependencies] 	github.com/cockroachdb/pebble@v0.0.0-20210302170351-6eee4f6b45b1: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
[15:54:24][Compile C dependencies] 	github.com/go-ole/go-ole@v1.2.2: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
[15:54:24][Compile C dependencies] 	github.com/klauspost/compress@v1.11.9: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
[15:54:24][Compile C dependencies] 	github.com/lightstep/lightstep-tracer-go@v0.15.6: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
[15:54:24][Compile C dependencies] 	golang.org/x/exp@v0.0.0-20210212053707-62dc52270d37: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
[15:54:24][Compile C dependencies] 	golang.org/x/sys@v0.0.0-20210217105451-b926d437f341: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
```

appears to me you may need to do another round of `make generate` / your vendor commit did not contain all changes necessary.

**Comment by cucaroach** [comment]
Thanks Oliver, it took me awhile but I finally found these errors in teamcity.   I'll see if I can repro them locally.

**Comment by cucaroach** [comment]
Okay I got the right changes pushed to vendor and its happier but there's some test failures I need to dig into.   Should I remove the reviewers until that's done?

**Comment by otan** [comment]
it's fine, just let us know when it's ready

File: go.mod [hunk 1]
```diff
@@ -157,13 +157,13 @@ require (
 	golang.org/x/text v0.3.5
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
 	golang.org/x/tools v0.1.0
-	google.golang.org/api v0.9.0
+	google.golang.org/api v0.13.0
```
Comment (otan) [inline go.mod:160]: can you revert these other changes in go.mod and see if `make vendor_rebuild` and `go mod tidy` are happy?
(just change cobra)
this usually happens using as `go get` tries to upgrade other things for you (but i don't know why).

**Comment by otan** [comment]
the vendor commit is also pretty gross with the fixup -- maybe worth just blowing it away and force pushing a rebased version

File: go.mod (hunk 1 above)
Comment (cucaroach) [reply go.mod:160]: Even if I just do s/0.0.5/1.1.3/ when vendor_rebuild.sh calls go mod vendor all these changes get re-made.

**Comment by cucaroach** [comment]
I got stuck on figuring how how to fix the -v flag, @knz saved the day with https://github.com/cockroachdb/cockroach/pull/63839

**Pull request closed without merging**